CMakeLists.txt

```diff
@@ -1604,6 +1604,11 @@ endif()
 hpx_libraries(${HWLOC_LIBRARIES})
 include_directories(${HWLOC_INCLUDE_DIR})
 
+################################################################################
+# Set search paths for generated headers.
+################################################################################
+include_directories("${CMAKE_BINARY_DIR}/include/generated")
```

Comment:
Befor @khuck's change for APEX we had just:
```
include_directories("${CMAKE_BINARY_DIR}")
```
so that HPX could find its own generated files (`hpx/config/defines.hpp`, etc.). How are those found now?

Reply:
Good observation. I moved the generated files to `include/generated` to have them sit alongside `include/libs` which held the symlinks to headers from our modules. But I've removed the symlinks so I could move the generated files back to where they were. @sithhell did you have some use case in mind with the symlinks that I've now broken? I've added the module include directories to the build-tree CMake and pkgconfig files which as far as I can see would've been the only use case for the symlinks.